bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -84,20 +84,30 @@
 <header id="masthead">
   <div class="container">
     {% block tabzilla_tab %}
-    <div id="tabzilla">
-      <a href="{{ url('mozorg.home') }}">mozilla</a>
-    </div>
+      <div id="tabzilla">
+        <a href="{{ url('mozorg.home') }}">Mozilla</a>
+      </div>
+    {% endblock %}
+
+    {% block site_header_nav %}
```

Comment:
Ah, nevermind - I see it still needs the `<div class="container>` which is different to the base template. We could probably refactor that to avoid this duplication, but can save that for another bug perhaps.